go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/fatih/structs v1.1.0
 	github.com/gertd/go-pluralize v0.1.1
+	github.com/ghodss/yaml v1.0.0
```

Comment:
It may be possible to use the `k8s.io/apimachinery/pkg/util/yaml` package, instead of introducing another dependency.